src/chapters/razor-views/bootstrap.rst

```diff
@@ -0,0 +1,80 @@
+Bootstrap
+=========
+
+Introduction
+------------
+
+Bootstrap is a commonly-used style library. It allows users to quickly apply its CSS style rules 
+with class selectors. Style updates can add features or improve the usability of an application. For example, 
+Bootstrap, and other styling libraries for that matter, use a standardized color scheme for items like clickable
+buttons. As the implementer of the helper library, you can apply the ``btn btn-primary`` classes to a button 
```

Comment:
Change "implementer" to "user". "Implementer" implies we're referring to the person _writing_ the helper library. (I fixed this in the Java book.)

Reply:
Fixed!